index.d.ts

```diff
@@ -0,0 +1,72 @@
+import * as stream from 'stream'
+import * as http from 'http'
+
+type HTTPMethods = 'DELETE' | 'GET' | 'HEAD' | 'PATCH' | 'POST' | 'PUT' | 'OPTIONS'
+
+declare namespace LightMyRequest {
+
+  type inject<Payload> = (
+    dispatchFunc: (req: Request, res: Response) => void,
+    options: string | InjectOptions,
+    callback: (err: Error, payload: any) => void | Promise<Payload>
+  ) => void
+
+  function isInjection (obj: Request | Response): boolean
+
+  interface InjectOptions {
+    url: string | {
```

Comment:
I’m not sure we can really pass an object here. Can you check in code?

Reply:
Based on the schema in `index.js`:
https://github.com/fastify/light-my-request/blob/f06c85f5c3f3dc98ac9adc37d7d0c2ad1cc5824a/index.js#L16-L27

Reply:
ping @mcollina for updates. Pinging just in case this notification slipped through; I know you can get very busy so no rush on this! Thank you 😁